app/services/calculate_subject_specialism_type.rb

```diff
@@ -10,6 +10,7 @@ def initialize(subjects:)
   def call
     return :primary if primary_subject?
     return :language if language_specialism?
+    return :language_and_other if any_language_specialism?
```

Comment:
can this simply be 
```suggestion
    return :language_and_other if any_subjects_are_modern_languages?
```
or perhaps just a comment to say `# i.e. any language specialisms?`